Microsoft.Toolkit.Uwp.UI.Controls/AdaptiveGridView/AdaptiveGridView.cs

```diff
@@ -82,10 +84,55 @@ protected override void OnApplyTemplate()
                 _listView.SizeChanged += ListView_SizeChanged;
                 _listView.ItemClick += ListView_ItemClick;
                 _listView.Items.VectorChanged += ListViewItems_VectorChanged;
+                _listView.SelectionChanged += ListView_SelectionChanged;
             }
 
             _isInitialized = true;
             OnOneRowModeEnabledChanged(this, OneRowModeEnabled);
+            InitializeBindings();
+        }
+
+        private void InitializeBindings()
+        {
+            // Set bindings from base control.
+            var selectedItemBinding = new Binding()
+            {
+                Source = this,
+                Path = new PropertyPath("SelectedItem"),
+                Mode = BindingMode.TwoWay
+            };
+
+            var isItemClickEnabledBinding = new Binding()
+            {
+                Source = this,
+                Path = new PropertyPath("IsItemClickEnabled"),
+                Mode = BindingMode.TwoWay
+            };
+
+            var selectionModeBinding = new Binding()
+            {
+                Source = this,
+                Path = new PropertyPath("SelectionMode"),
+                Mode = BindingMode.TwoWay
+            };
+
+            var selectionIndexBinding = new Binding()
+            {
+                Source = this,
+                Path = new PropertyPath("SelectedIndex"),
+                Mode = BindingMode.TwoWay
+            };
+
+            _listView.SetBinding(ListViewBase.SelectionModeProperty, selectionModeBinding);
+            _listView.SetBinding(ListViewBase.IsItemClickEnabledProperty, isItemClickEnabledBinding);
+            _listView.SetBinding(Selector.SelectedItemProperty, selectedItemBinding);
+            _listView.SetBinding(Selector.SelectedIndexProperty, selectionIndexBinding);
+        }
```

Comment:
You're right about OneWay bindings and it looks like SelectedIndex and SelectedItem properties can't be done with TemplateBinding. On the other hand IsItemClickEnabled and SelectedItems properties look fine without initializing them.

I will update the code and send a new commit soon, thanks for pointing out.